src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -51,13 +52,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
 
+    String place_id = request.getParameter("place_id");
     String userEmail = userService.getCurrentUser().getEmail(); // Used to restrict user to one review/location
-    String newReview = request.getParameter("comment");
+    String reviewText = request.getParameter("comment");
+    Date time = request.getParameter("timeStamp");
     String firstName = request.getParameter("firstname");
     String lastName = request.getParameter("lastname");
-    String rating = request.getParameter("rate");
-    String place_id = request.getParameter("place_id");
+    double rating = request.getParameter("rate");
+
+    Comment newReview = new Comment(userEmail, newReview, time);
+    PlaceReviews curLocation = new PlaceReviews(); // TODO grab current location by ID
+
+    if (curLocation != null) {
+      // TODO: Add or modify current review
+    } else {
+      // TODO: Create new PlaceReviews and initialize
+    }
```

Comment:
I would much rather you have the TODO comments without the unused variable and empty if/else statements. It can be confusing to a reader and adds noise to the code.